tests/missing_refs.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+PYTHON_EXECUTABLE="$1"
+BENCHMARK="$2"
+PROFILES="$3"
```

Comment:
this is a risk of quoting problems

what about making every argument starting from 3 a profile? eg. use shift

Reply:
Aha, that's much better approach. Thanks, I will update it